cmd/monaco/test-resources/integration-all-configs/project/service-detection-full-web-request/demorule.json

```diff
@@ -0,0 +1,40 @@
+{
+  "type": "FULL_WEB_REQUEST",
+  "managementZones": [],
+  "name": "{{ .name }}",
+  "description": "{{ .name }}",
+  "enabled": {{ .enabled }},
+  "conditions": [
+    {
+      "attributeType": "SERVER_NAME",
+      "compareOperations": [
+        {
+          "type": "EQUALS",
+          "negate": false,
+          "ignoreCase": false,
+          "values": [
+            "demoserver1",
+            "demoserver2"
+          ]
+        }
+      ]
+    },
+    {
+      "attributeType": "URL_PATH",
+      "compareOperations": [
+        {
+          "type": "EXISTS",
+          "negate": false
+        }
+      ]
+    }
+  ],
+  "applicationId": null,

```

Comment:
_nitpick:_ 
the `null` values here and in other test configs are a bit odd, especially if someone uses them as a sample.
could we put some dummy values in?

Reply:
The `applicationId` is optional in the model. I'll remove it.